libs/components/src/lib/modal-chooser-db/modal-chooser-db.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import Modal from 'react-modal';
+import { ScholarsDB } from '@foodmedicine/interfaces';
+
+const customStyles = {
+  content: {
+    top: '50%',
+    left: '50%',
+    right: 'auto',
+    bottom: 'auto',
+    marginRight: '-50%',
+    transform: 'translate(-50%, -50%)',
+  },
+};
+
+interface ModalChooseProps {
+  onClose: (db: ScholarsDB) => unknown;
+  initialDB: ScholarsDB;
+}
+
+Modal.setAppElement('#root');
+
+export function ModalChooserDB(props: ModalChooseProps) {
+  const [modalIsOpen, setIsOpen] = useState(false);
+  const [db, setDB] = useState(props.initialDB);
+  function openModal() {
+    setIsOpen(true);
+  }
+
+
+  function closeModal() {
+    setIsOpen(false);
+    props.onClose(db);
+  }
+
+  return (
+    <div>
+      <button onClick={openModal}>Select a scholarly database</button>
+      <Modal
+        isOpen={modalIsOpen}
+        onRequestClose={closeModal}
+        style={customStyles}
```

Comment:
prefer to have this Modal have an attached css class, and the style be defined in a .css file